sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,104 @@
+
+"In progress" teams have started localization work, but are not yet live on the Kubernetes site.
+
+## Localization team structure
+
+Anyone is open to start a new localization effort for any language or join an ongoing localization team. If you are interesting in learning more about specific localization requirements or processes see the [Localization Contributing Guide](https://kubernetes.io/docs/contribute/localization/).  
+
+Localization team are self sufficient communities within SIG Docs. Localization teams are responsible for
+
+- Localizing content on kuberntes.io into their chosen language.  
+- Timely reviews and merging PRs for their languages according to the [review and merge prcesses](https://kubernetes.io/docs/contribute/review/) of the kubernetes/website repo.
+- Building a community that can support their localization efforts. Note you need at least two contributors to begin a localization team because contributors can't approve their own pull requests. You will likely need more contributors, reviewers, and approvers to create a sustainable localization effort.
+- Organizing and prioritizing work to localize content.
+- Understanding and following the [Kubernetes Code of Conduct](https://kubernetes.io/community/code-of-conduct/) within their teams. Questions or concerns around Code of Conduct violations should be raised to the SIG Docs, Localization Subproject leadership, or [Kubernetes Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct). 
+
+The Localization subproject leads and the SIG Docs leadership are always available to help localization teams by answering questions and providing feedback.
+
+### Localization team leads
+
+A leader of a localization team is someone who is responsible for helping to guide the localization efforts for a given language. Its recommended that each localization team has at least 2 leads.   
+
+Some recommended skills for localization leads include, 
+* Excellent written and verbal skills in the given language.
+* Experience with the kubernetes/website repo localization [branching strategy](https://kubernetes.io/docs/contribute/localization/#branching-strategy).
+* Experience reviewing English language PRs to develop an understanding the review processes and repo structure. Its recommended that leaders participate in the SIG Docs [PR Wrangler Shadow Program](https://kubernetes.io/docs/contribute/participate/pr-wranglers/#pr-wrangler-shadow-program).
+
+
+## Localization processes
+
+See the [Localization Contributing Guide](https://kubernetes.io/docs/contribute/localization/) for full information on localization processes and policies. This page includes information on [starting a new localization](https://kubernetes.io/docs/contribute/localization/#start-a-new-localization), [branching strageies](https://kubernetes.io/docs/contribute/localization/#branching-strategy) for localizing content, and more. 
+
```

Comment:
We'd probably not want all these extra lines at the end.

Reply:
updated